static/openapi/prerendering-spec.json

```diff
@@ -1496,29 +809,1153 @@
+                                "isActive": {
+                                  "type": "boolean",
+                                  "description": "Whether the specific interaction is active or not."
+                                },
+                                "htmlSelector": {
+                                  "type": "string",
+                                  "description": "HTML selector where the interaction is executed."
+                                },
+                                "waitAfterInteraction": {
+                                  "type": "integer",
+                                  "description": "Time to wait (in seconds) after the interaction execution."
+                                },
+                                "executionSequence": {
+                                  "type": "integer",
+                                  "description": "Specific interaction execution order."
+                                },
+                                "urlPaths": {
+                                  "type": "array",
+                                  "items": {
+                                    "type": "string",
+                                    "description": "Origin path"
+                                  },
+                                  "description": "Origin paths where the specific interaction is executed or not, depending on the \"operation\" attribute."
+                                },
+                                "operation": {
+                                  "type": "string",
+                                  "enum": ["==", "!="],
+                                  "description": "Operator applied against the \"urlPaths\" list."
+                                }
+                              },
+                              "required": [
+                                "isActive",
+                                "htmlSelector",
+                                "executionSequence",
+                                "urlPaths",
+                                "operation"
+                              ]
+                            }
+                          }
+                        },
+                        "required": ["isActive", "global", "specific"]
+                      }
+                    }
+                  }
+                ]
+              }
+            }
+          }
+        },
+        "parameters": [
+          {
+            "schema": { "type": "string", "minLength": 1 },
+            "in": "path",
+            "name": "origin",
+            "required": true,
+            "description": "Origin hostname."
+          },
+          {
+            "schema": {
+              "type": "string",
+              "enum": ["hover", "click", "scroll"]
+            },
+            "in": "path",
+            "name": "type",
+            "required": true,
+            "description": "Interaction type (scroll, click, hover, etc...)"
```

Comment:
List the interaction types, in all cases. Something like, "Options are: scroll, click, and hover."

The "etc." implies there are more that are not listed.

Reply:
The idea is to provide some examples, not all options. Otherwise, we may have problems keeping this list updated.
We can update it to `Interaction type (e.g., scroll, click, hover)`

Reply:
We can also remove the examples, as all options appears in the documentation page:
<img width="232" alt="image" src="https://github.com/Macrometacorp/docs/assets/148809620/bcc4d719-390d-4b25-9f99-6aad12838d4a">

Any thoughts?

Reply:
As long as the user only has to go to one page to see all of this information, it's good. I didn't realize the values also appeared.

Reply:
Yeah, they appear when you mark the field as an enum and provide all options.